app/views/rating_questions/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="form">
+  <div class="flash-notice"><%= flash[:notice] %>
+  </div>
+  <h4 class="new_question"><%= link_to "New Question", new_rating_question_path %></h4>
```

Comment:
This line might be more easily read if it was split out:

```
<h4 class="new_question">
  <%= link_to "New Question", new_rating_question_path %>
</h4>

Reply:
100% agreed.